### ISSUE TITLE
Can't Properly Configure

### ISSUE THREAD

**Issue by NickSchweitzer** [SpoonX/aurelia-charts#23]
I'm using the SPA Templates for ASP.NET Core as the base of an application
https://github.com/aspnet/JavaScriptServices

I created a base application and have been making additions to it. I installed Aurelia-charts according to the directions for Webpack in the instructions, but am confused by the following line:
"And add aurelia-charts in the coreBundles.aurelia section of your webpack.config.js"

There is no coreBundles.aurelia section in my webpack.config file (attached with a .txt extension) and I can't seem to find documentation on where it should go, or what the properties are. Can you clarify that? 

When I add the .plugin calls described to the configure function, it basically breaks my app from loading. If I comment out plotly and charts, it still breaks. The only one that seems to work without stopping the app from loading is c3.
        .plugin('aurelia-charts-plotly')
        .plugin('aurelia-charts-c3')
        .plugin('aurelia-charts', charts => {
            charts.configure({
                defaults: {
                    /* the default library type */
                    library: 'C3',
                    /* the default chart type */
                    type: 'line',

                    /* what library to use for the chart type */
                    libraries: {
                        bar: 'c3',
                        geo: 'datamaps'
                    }
                }
            })
        });
[webpack.config.js.txt](https://github.com/SpoonX/aurelia-charts/files/936627/webpack.config.js.txt)

**Comment by NickSchweitzer** [comment]
After some further tweaking (adding some missing dependencies), I'm still running into the same issues, but have some further questions. In the Output window as I run the app, I get the following:

Microsoft.AspNetCore.NodeServices:Error: Unable to resolve component/dimensions-picker.html
Microsoft.AspNetCore.NodeServices:Error: Unable to resolve aurelia-charts/component/chart-picker.
Microsoft.AspNetCore.NodeServices:Error: Unable to resolve aurelia-charts/component/dimensions-picker

Where can I add those dependencies so they get picked up? That might resolve my issue

**Comment by RWOverdijk** [comment]
@bas080 could you help this guy out? :)

**Comment by NickSchweitzer** [comment]
Haven't heard anything - I've dropped this plugin in favor of aurelia-chart which I was able to get to work based on their instructions.

**Comment by RWOverdijk** [comment]
@NickSchweitzer 👍  keeping this open though

**Comment by JonSomNor** [comment]
Could someone please provide an answer to Nicks question??
I'm probably not the only one looking for the coreBundles.aurelia section of webpack.config.js...

**Comment by bas080** [comment]
I have not used webpack together with this plugin. I'm unlikely able to help you anytime soon @NickSchweitzer and @JonSomNor.

**Comment by JonSomNor** [comment]
Ok, thanx for responding anyway bas080!

**Comment by robscottnh** [comment]
I'm having the same issue.  There is no coreBundles.aurelia seciton of webpack.config.js when using the asp.net core templates.  What should it look like and where should it go?

**Comment by ehud-swiftstack** [comment]
same here. no coreBundles

**Comment by ehudkaldor** [comment]
same here. no coreBundles

**Comment by robscottnh** [comment]
I've figured out a little bit of the issue, I'm not exactly sure how to overcome it (WebPack newb), but I've outlined what I think is going on here:

[https://github.com/aspnet/JavaScriptServices/issues/1406](url)

TLDR: coreBundles is the name of string that is used in one version of webpack.config.js (that is used by the developers of many of these plug-ins), while the webpack.config.js generated by the ASP.NET Core skeletons doesn't use that string, but seems to be using another method to gather the dependencies.

Being a newb to webpack, I'm not sure how to resolve the two methods of describing the dependencies.

Regards,
Rob

**Comment by RWOverdijk** [comment]
PRs are welcome. Anyone willing to step forward to help maintain this lib can contact me, too.
